src/main/resources/locale_pt_PT.properties

```
@@ -0,0 +1,196 @@
+command.reputation.description=Informação sobre a sua reputação ou sobre a reputação de outros.
+command.reputation.profile.nextLevel=Próximo nível
+command.reputation.profile.title=Reputação de %NAME%
+command.reputation.sub.top=Abter a lista do topo ou uma página da lista
+command.reputation.sub.top.ranking=Ranking de %GUILD%
+command.roles.description=Gerir papéis de reputação.
+command.roles.sub.add=Adicionar um papel de reputação.
+command.roles.sub.add.added=Added role %ROLE% com reputação %POINTS%.
+command.roles.sub.list=Listar papéis.de reputação.
+command.roles.sub.managerRole=Alterar o papel de gerente. Os utilizadores com esta função podem gerir o bot.
+command.roles.sub.managerRole.current=A actual função de gestor é %ROLE%.
+command.roles.sub.managerRole.noRole=Sem função de gestor definida.
+command.roles.sub.managerRole.set=A função de gestor definida como %ROLE%.
+command.roles.sub.remove=Remover uma função de reputação.
+command.roles.sub.remove.notARepRole=Esta função não é uma função de reputação.
+command.roles.sub.remove.removed=Remover um papel %ROLE%.
+command.scan.canceled=Scan cancelado.
+command.scan.canceling=Tentativa de cancelar o scan.
+command.scan.completed=Varredura concluída.
+command.scan.description=Scan a channel for reputations.
+command.scan.error.history=Desaparecimento da permissão para histórico de mensagens
+command.scan.error.queueFull=Todos os scans estão actualmente em execução. Por favor aguarde até que uma ranhura esteja livre.
+command.scan.error.running=Já está a decorrer uma tarefa nesta guild. Por favor aguarde até que a tarefa esteja terminada.
+command.scan.progress=Progress: %PERCENT%
+command.scan.result=Scanned %SCANNED% messages.\nAdded %HITS% reputation.
+command.scan.scheduling=Agramação do seu scan. O tempo necessário estimado até à conclusão é: %DURATION%
+command.scan.sub.cancel=Cancelar um scan em execução.
+command.scan.sub.cancel.noTask= Nenhuma tarefa está em execução para esta guilda.
+command.setup.description=Executar uma configuração básica do bot.
+command.setup.dialog.channels=Por favor introduza todos os canais onde os utilizadores têm permissão para receber reputação.
+command.setup.dialog.continueToProceed=Please press "$word.continue$" to proceed.
+command.setup.dialog.loadDefaults=Carregar palavras de agradecimento por defeito para as suas línguas.
+command.setup.dialog.managerRole=Please write one role which should be allowed to change the settings of the bot.
+command.setup.dialog.roles=Por favor defina alguns papéis de reputação.\nEstes papéis serão dados a um utilizador quando este atingir a quantidade especificada de reputação.\nDefina-os como <$words.role$> <$words.reputation$>.\nQuando tiver definido todos os seus papéis prima o botão.
+command.setup.dialog.rolesFormat=Por favor defina-os como <$words.role$> <$words.reputation$>
+command.setup.dialog.selectLanguage=Please select your language.
+command.setup.dialog.setupComplete=Setup complete!
```

Looks untranslated

DeepL suggestion:
```suggestion
command.setup.dialog.setupComplete=Configuração completa!
```